tests/test_georaster.py

```diff
@@ -137,9 +122,9 @@ def test_loading(self) -> None:
         assert r.height == 655
         assert r.shape == (r.height, r.width)
         assert r.count == 1
-        assert geoutils.misc.array_equal(r.dtypes, ["uint8"])
+        assert np.array_equal(r.dtypes, ["uint8"])
```

Comment:
I guess this series of test was copy/pasted from similar ones above that use `array_equal` because there is more than 1 `dtype` item. Fine to leave it like this.